include/openssl/engine.h

```diff
@@ -1160,6 +1160,7 @@ ENGINE_FUNC(ENGINE *, EVP_PKEY_get0_engine, (const EVP_PKEY *pkey), NULL)
 /* ENGINE *DH_get0_engine(DH *d); */
 ENGINE_FUNC(ENGINE *, DH_get0_engine, (DH * d), NULL)
 
+#ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
The other functions which does not originate from `engine.h` and were deprecated earlier should be guarded with the ifdef too, otherwise they'll appear as a symbol even they were turned off.

Reply:
yes, fixed. Also removed one function that was internal and should not be here at all.